File: RS.VueClient/src/Models/ImgVerifyResultModel.ts
import type { RectModel } from "./WebAPI/RectModel";

/**
 * 验证码结果类
 */
export class ImgVerifyResultModel {


  /**
  * 验证矩形框
  */
  public Verify: RectModel | null = null;


  /**
  * 验证会话Id
  */
  public VerifySessionId: string | null = null;

}

File: RS.Models/ImgVerifyResultModel.cs
using System;
using System.Collections.Generic;
using System.Drawing;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace RS.Models
{
    public class ImgVerifyResultModel
    {
        /// <summary>
        /// 验证矩形框
        /// </summary>
        public RectModel? Verify { get; set; }

        /// <summary>
        /// 验证会话Id
        /// </summary>
        public string? VerifySessionId { get; set; }

    }
}
